src/easylink/pipeline_schema_constants/development.py

```diff
@@ -1,24 +1,89 @@
 from easylink.graph_components import Edge, InputSlot, OutputSlot, SlotMapping
-from easylink.step import BasicStep, HierarchicalStep, InputSlot, IOStep, LoopStep
+from easylink.step import (
+    BasicStep,
+    HierarchicalStep,
+    InputSlot,
+    IOStep,
+    LoopStep,
+    ParallelStep,
+)
 from easylink.utilities.validation_utils import validate_input_file_dummy
 
 NODES = [
-    IOStep(step_name="input_data", input_slots=[], output_slots=[OutputSlot("file1")]),
-    BasicStep(
+    IOStep(step_name="input_data", input_slots=[], output_slots=[OutputSlot("all")]),
+    ParallelStep(
         step_name="step_1",
         input_slots=[
             InputSlot(
                 name="step_1_main_input",
                 env_var="DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS",
                 validator=validate_input_file_dummy,
             ),
-            InputSlot(
-                name="step_1_secondary_input",
-                env_var="DUMMY_CONTAINER_SECONDARY_INPUT_FILE_PATHS",
-                validator=validate_input_file_dummy,
-            ),
         ],
         output_slots=[OutputSlot("step_1_main_output")],
+        split_node=HierarchicalStep(
```

Comment:
`split_node` is not a super intuitive name to me -- anything else you considered?

Reply:
how do you feel about "template_step" or something like that, suggestive of being a single step that is copied to create the full graph (this could apply to loop step as well)

Reply:
I like that a lot better! I think the idea of a "template" for both loops and parallel sections is more intuitive.